lib/policyEvaluator/utils/conditions.ts

```diff
@@ -279,9 +277,9 @@ export function convertConditionOperator(operator: string): boolean {
             } else {
                 return policyValRegex(key);
             }
+            return true;
```

Comment:
I think we should either return undefined here, because if we reach this stage, it means the "key" was an array and a prefix that is unknown...